js/read_dir_test.ts

```diff
@@ -13,7 +15,11 @@ function assertSameContent(files: FileInfo[]): void {
     }
 
     if (file.name === "002_hello.ts") {
-      assertEquals(file.path, `tests/${file.name}`);
+      if (isWin) {
+        assert(file.path.endsWith(`tests\\${file.name}`));
+      } else {
+        assert(file.path.endsWith(`tests/${file.name}`));
```

Comment:
Odd that this changed... Do you know why?

Reply:
Had to make some changes to file path parsing, so I could pass the same full absolute path to permissions checks. It turns out that there isn't a standard way to parse paths other `canonicalize()`, but that requires that the path exists and replaces symbolic links with their destination(making it impossible to stat a symbolic link directly).